src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -95,7 +107,7 @@ public void save(AddressBook addressBook) throws StorageOperationException {
             final Marshaller marshaller = jaxbContext.createMarshaller();
             marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
             marshaller.marshal(toSave, fileWriter);
-
+            isFileExist = true;
```

Comment:
Did you test to confirm this works? Looks like the exception will never be thrown.